guest-agents/qemu-guest-agent/qemu-guest-agent.yaml

```diff
@@ -20,7 +19,7 @@ container:
         - bind
         - ro
     # State files.
-    - source: /var/run
+    - source: /system/run/qemu-guest-agent
```

Comment:
does it need any state to be actually mounted from the host? given it's tmpfs, it just preserves state across service restarts, not host restarts (?)

Reply:
i don't think so, just needs a writable `/var/run`

Reply:
I just wonder if we need support for `emptyDir` (or some equivalent of it) mounts...

Reply:
this might fit well into our volumes ideas

Reply:
well, in this case it's per-container (or even per-container run) temporary mount, we could do same thing as kubelet does - create some temporary directory under `/system`  or `/tmp`, mount it up into the container, and remove the directory when the container stops?

Reply:
on other hand, it probably makes sense to backport it _this way_ to 1.6.x, and we can work on `emptyDir` mounts next

Reply:
> on other hand, it probably makes sense to backport it this way to 1.6.x, and we can work on emptyDir mounts next

yeh, let's get this merged in as and once we have emptyDir support, let's update